google/ggrc-core
# GGRC-5826,GGRC-5223 Add 'Attachments' field for Programs, Scope objects, Core GGRC Objects, Issues

# Issue description

**GGRC-5826:**
* When we map document to any object with defined folder_id, the document should be added to the folder.

**GGRC-5223:**
* Add 'Attachments' field for Programs, Scope objects, Core GGRC Objects, Issues

# Steps to test the changes

* Create an object from the following:
    * Control
    * Issue
    * AccessGroup
    * Clause
    * Contract
    * DataAsset
    * Facility
    * Market
    * Metric
    * Objective
    * OrgGroup
    * Policy
    * Process
    * Product
    * ProductGroup
    * Program
    * Project
    * Regulation
    * Requirement
    * Risk
    * Standard
    * System
    * TechnologyEnvironment
    * Threat
    * Vendor
* Attach gdrive folder to this object
* Attach document to this object
* Check gdrive folder
**Expected result:** document file is appeared in gdrive folder

# Solution description

**BE:**
* ~~Add Folderable objects to the ALLOWED_PARENTS for Document.~~
* Change document parent validation to avoid adding new classes to ALLOWED_PARENTS
* Add test of adding a file from mapped document to object's gdrive folder.

**FE:**
* Use for all required objects the same components as for Control.

# Sanity checklist




# PR Review checklist
